src/kafka/kafka-server.ts

```diff
@@ -21,7 +21,8 @@ const defaultConfig: Partial<KafkaConfiguration> = {
   rdConfig: {},
   connectionTimeout: 1000,
   flushTimeout: 2000,
-  initialOffset: InitialOffset.latest
+  initialOffset: InitialOffset.latest,
+  projectName: process.env.npm_package_version
```

Comment:
`process.env.npm_package_version` is either string or undefined, so I'd add another default, something like:
```ts
process.env.npm_package_version || 'unnamed-project'
```